public/locales/en/register.json

```diff
@@ -182,7 +182,14 @@
     },
     "transactions": {
       "heading": "Almost there",
-      "subheading": "You will need to complete two transactions to secure your name. The second transaction must be completed within 24 hours of the first.",
+      "subheading": {
+        "default": "You will need to complete two transactions to secure your name. The second transaction must be completed within 24 hours of the first.",
+        "commiting": "This wait prevents others from <tooltip>front running</tooltip> your transaction. You will be prompted to complete a second transaction when the timer is complete.",
```

Comment:
<img width="780" alt="image" src="https://github.com/user-attachments/assets/666a04f6-7945-492b-b4cc-7c0f3bda1505">
The link is returning 404